docs/UserGuide.md

```diff
@@ -277,15 +322,15 @@ Examples:
 ### Saving the data
 [<sub><sup>Back to top</sup></sub>](#table-of-contents)
 
-TAilor data are saved in the hard disk automatically after any command that changes the data.
-There is no need to save manually.
+TAilor's data is saved in the hard disk automatically after any command that changes the data.
+There is no need to save it manually.
 
 ### Editing the data file
 [<sub><sup>Back to top</sup></sub>](#table-of-contents)
 
-TAilor data are saved as a JSON file `[JAR file location]/data/contactlist.json`.
-Task List data is also saved as a JSON file `[JAR file location]/data/tasklist.json`. Advanced users are
-welcome to update the data directly by editing those data files.
+TAilor's data is saved as two JSON files. The first being ContactList's data as `[JAR file location]/data/contactlist.
```

Comment:
3 Json files now, moduleList is also a JSON file.

Reply:
I will be adding the json files part in my UG PR later.